### ISSUE TITLE
Openshift deployment not working

### ISSUE THREAD

**Issue by corporationsruleyourmom** [angular-fullstack/generator-angular-fullstack#1988]
Hey, big fan of the generator! Thanks for all your hard work!

I can't deploy on Openshift or anywhere really. I've used ``yo angular-fullstack:openshift`` and ``grunt buildcontrol:openshift`` but neither have worked successfully. I know this issue has todo with the default Node cartridge (v0.10) that is used when deploying to Openshift. I believe with the new Openshift Origin which is supposed to be similar to Docker will eliminate this deployment issue when it is fully launched. I've also checked out https://hub.docker.com/r/generatorangularfullstack/angular-fullstack-dist/ to see if I can set up a Docker but I was unable to get that deployment method to work either. Any help deploying would be greatly appreciated!

Item | Version
----- | -----
generator-angular-fullstack | 2.x and 3.x
Node | 0.12 and 4.x
npm | 3.8.2
Operating System | linux

Item | Answer
----- | -----
Transpiler | Babel
Markup | HTML 
CSS | CSS 
Router | ngRoute
Build Tool | Grunt
Client Tests | Jasmine
DB | MongoDB
Auth | Y

**Comment by Awk34** [comment]
Yeah, we really don't support Node 0.12 anymore, let alone 0.10. Hopefully this can all get resolved with the Openshift Docker infrastructure you mentioned! I've never used Openshift myself, so I really depend on other community members to help with these areas of the generator..

**Comment by Awk34** [comment]
Also be aware that the [openshift doc](https://github.com/angular-fullstack/generator-angular-fullstack/blob/master/docs/generators/openshift.md) has a link about upgrading from old node versions
